### DISCUSSION TITLE
Display prompts underneath picture stimuli

### DISCUSSION THREAD

**Discussion by ghost** [jspsych/jsPsych#838] [Q&A] [answered]
Dear all,

When I tried to add a prompt underneath the picture (which i think is the default setting), it appears next to the picture instead.

I am wondering what is wrong with this, as i need the prompt to be under the picture.

`
<img width="517" alt="WechatIMG216" src="https://user-images.githubusercontent.com/67168823/86589821-49a08180-bf86-11ea-8496-c0705da49592.png">
<img width="790" alt="WechatIMG217" src="https://user-images.githubusercontent.com/67168823/86589825-4a391800-bf86-11ea-8999-6dd7a50eebaa.png">

`
Best,
Chiu-Chou

**Comment by jodeleeuw** [comment, marked as answer]
Hi @ChiuChouH 

I'm not able to replicate this problem. Do you have the jspsych.css file loaded? Do you have any custom CSS rules defined? What browser are you using to run the experiment?

**Reply by ghost** [reply]
Hi there,

Thank you for your reply. I have worked it out. Nothing has been changed in the CSS file and the experiment was run in Chrome. It is actually quite weird that it worked when I copied an pasted all the codes to a new file (while the old file still has the problem).

**Reply by jodeleeuw** [reply]
Odd. Well, I'm glad it is resolved!
